lg69t/README.md

```diff
@@ -16,21 +16,22 @@ python3 -m pip install -r requirements.txt
 
 And download the latest firmware package from [Point One's Developer Portal](https://pointonenav.com/docs/).
 
-
 ## Application and GNSS
 
-To update the Application and GNSS firmware:
+To update the Application and GNSS firmware from previous Point One Firmware, you must be use the Standard COM Port (UART1, typically /dev/ttyUSB1 in Linux for P1SDK):
```

Comment:
The bootloader should support programming from either UART. Is that not working?

Reply:
nope, but maybe future!

Reply:
no need to say "from Previous P1 Firmware". Also grammar: must use, not must be use.